.github/workflows/web-test-e2e.yml

```diff
@@ -19,13 +19,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [18.x]
+        node-version: ['20']
```

Comment:
Looks like we will need to update it when your second PR is merged.

Reply:
I rebased the changes. Awaiting the outcome of the tests now..